echemdb/cv/cv_entry.py

```diff
@@ -0,0 +1,236 @@
+r"""
+A Data Package describing a Cyclic Voltammogram.
+
+These are the individual elements of a :class:`CVDatabase`.
+
+EXAMPLES:
+
+Data Packages containing published data,
+also contain information on the source of the data::
+
+    >>> from echemdb.cv.cv_database import CVDatabase
+    >>> db = CVDatabase.create_example()
+    >>> entry = db['alves_2011_electrochemistry_6010_f1a_solid']
+    >>> entry.bibliography  # doctest: +NORMALIZE_WHITESPACE +REMOTE_DATA
+    Entry('article',
+      fields=[
+        ('title', 'Electrochemistry at Ru(0001) in a flowing CO-saturated electrolyte—reactive and inert adlayer phases'),
+        ('journal', 'Physical Chemistry Chemical Physics'),
+        ('volume', '13'),
+        ('number', '13'),
+        ('pages', '6010--6021'),
+        ('year', '2011'),
+        ('publisher', 'Royal Society of Chemistry'),
+        ('abstract', 'We investigated ...')],
+      persons=OrderedCaseInsensitiveDict([('author', [Person('Alves, Otavio B'), Person('Hoster, Harry E'), Person('Behm, Rolf J{\\"u}rgen')])]))
+
+"""
+# ********************************************************************
+#  This file is part of echemdb.
+#
+#        Copyright (C) 2021-2022 Albert Engstfeld
+#        Copyright (C)      2021 Johannes Hermann
+#        Copyright (C) 2021-2022 Julian Rüth
+#        Copyright (C)      2021 Nicolas Hörmann
+#
+#  echemdb is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  echemdb is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with echemdb. If not, see <https://www.gnu.org/licenses/>.
+# ********************************************************************
+import logging
+
+from echemdb.entry import Entry
+
+logger = logging.getLogger("echemdb")
+
+
+class CVEntry(Entry):
+    r"""
+    A [data package](https://github.com/frictionlessdata/datapackage-py)
```

Comment:
This link won't render correctly https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks